cbqueryx/preparedquery_test.go

```diff
@@ -203,5 +205,7 @@ func TestPreparedQueryPreparedNameMissing(t *testing.T) {
 
 	assertQueryResult(t, expectedRows, &expectedResult, res)
 
-	require.Empty(t, cache.queryCache)
+	stmt, ok := cache.Get("SELECT 1=1")
```

Comment:
What's this change doing?

Reply:
queryCache is no longer accessible. This is looking up the statement in the cache to ensure it isn't in there.